README.md

```diff
@@ -26,9 +31,17 @@ For a complete list of available packages, please see the [latest available pack
 
 Last stable versions of packages that are production-ready. These libraries provide similar functionalities to the preview packages, as they allow you to use and consume existing resources and interact with them, for example: upload a storage blob. Stable library directories typically contain 'Microsoft.Azure' in their names, e.g. 'Microsoft.Azure.KeyVault'. They might not implement the [guidelines](https://azure.github.io/azure-sdk/dotnet_introduction.html) or have the same feature set as the Novemeber releases. They do however offer wider coverage of services.
 
-### Management
+### Management: New Releases
+A new set of management libraries that follow the [Azure SDK Design Guidelines for .NET](https://azure.github.io/azure-sdk/dotnet_introduction.html) are now in Public Preview. These new libraries provide a number of core capabilities that are shared amongst all Azure SDKs, including the intuitive Azure Identity library, an HTTP Pipeline with custom policies, error-handling, distributed tracing, and much more. You can find the list of new packages [on this page](https://azure.github.io/azure-sdk/releases/latest/dotnet.html). 
 
-Libraries which enable you to provision specific server resources. They are directly mirroring Azure service's REST endpoints. Management library directories typically contain the word 'Management' in their names, e.g. 'Microsoft.Azure.Management.Storage'.
+To get started with these new libraries, please see the [quickstart guide here](https://github.com/Azure/azure-sdk-for-net/blob/master/doc/mgmt_preview_quickstart.md)
+
+> NOTE: If you need to ensure your code is ready for production use one of the stable, non-preview libraries.
+
+### Management: Previous Versions
+For a complete list of management libraries which enable you to provision and manage Azure resources, please check [here](https://azure.github.io/azure-sdk/releases/latest/all/dotnet.html). They might not have the same feature set as the new releases but they do offer wider coverage of services. Management libraries can be identified by namespaces that start with `Azure.Management`, e.g. `Azure.Management.Network`
+
+Documentation and code samples for these libraries can be found [here](https://azure.github.io/azure-sdk-for-net).
```

Comment:
also should we mention that the transition plan such as "As we releasing new versions of SDKs for an Azure services, any existing management SDK (Microsoft.Azure.Management.XX) will only be in maintanence mode and be deprecated in the future inline of the standard Microsoft deprecation schedule, .....

Reply:
the official deprecation announcement will come later, once we are confident with the shape of what track 2 .NET should look like